events-api/src/modules/events/dto/requests/list-events-params.dto.ts

```diff
@@ -1,5 +1,7 @@
+import { SortOrderEnum } from './../../types/query.type';
 import { Transform } from 'class-transformer';
-import { IsInt, IsOptional, IsPositive, IsString, Max } from 'class-validator';
+import { IsEnum, IsInt, IsOptional, IsPositive, IsString, Max } from 'class-validator';
+import { EventSortEnum } from '../../types/query.type';
```

Comment:
Enum should not be stored in types. Suggest to store them in constants

Reply:
Understood and corrected